tests/e2e/specs/miscellaneous/ExtensionActivationTest.spec.ts

```diff
@@ -37,7 +38,12 @@ suite(`Extension Activation Test ${BASE_TEST_CONSTANTS.TEST_ENVIRONMENT}`, funct
 	const testWorkspaceUtil: ITestWorkspaceUtil = e2eContainer.get(TYPES.WorkspaceUtil);
 	const workspaces: Workspaces = e2eContainer.get(CLASSES.Workspaces);
 
-	const FACTORY_URL: string = 'https://github.com/crw-qe/python-hello-world/tree/test-vscode-extensions';
+	const factoryUrl: string = BASE_TEST_CONSTANTS.IS_CLUSTER_DISCONNECTED()
```

Comment:
It would be safer to put the test repo into the https://gh.crw-qe.com/test-automation-only/ organization.